test/storage/garbage_collector_test.cpp

```diff
@@ -1394,16 +1408,16 @@ TEST_F(GarbageCollectorTests, DeleteUpdateTest2) {
   }
 }
 
+/*
+ * Consider performing GC on the version chain sorted from newest to oldest:
+ * DELETE
+ * txn1 <- active txn
```

Comment:
Consistency with the previous notation
```suggestion
 T1 <- active txn
```